pkg/client/http.go

```diff
@@ -41,7 +41,10 @@ func NewClient(baseURL string, opts ...Option) (*Client, error) {
 	}
 	if options.ignoreTLSCert {
 		c.HTTPClient.Transport = &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+			TLSClientConfig:     &tls.Config{InsecureSkipVerify: true},
+			MaxIdleConns:        500,
```

Comment:
These values should be passed in via options (use these as defaults, but allow for changing them)